react/Price.js

```diff
@@ -67,9 +71,10 @@ class Price extends Component {
                   }}
                 />
               ) : (
-                <span>{installments}X {formattedInstallmentPrice}</span>
-              )
-              }
+                <span>
+                  {installments}X {formattedInstallmentPrice}
```

Comment:
Try to use the unicode value for `times` https://stackoverflow.com/questions/12009545/what-is-the-proper-html-entity-for-the-x-in-a-dimension

Reply:
I made the change, but what about the locales messages? They will be different

Reply:
You can pass the span element containing the Unicode as a parameter to the locale text.